.storybook/stories/AuthenticatedAppShell/AuthenticatedAppShell.test.tsx

```diff
@@ -0,0 +1,71 @@
+import '@testing-library/jest-dom';
+import 'html-validate/jest';
+import { cleanup, render } from '../../../test-utils';
+import { AuthenticatedAppShell } from './AuthenticatedAppShell';
+import {
+	ChartIcon,
+	FactoryIcon,
+	HomeIcon,
+	MailIcon,
+	QuestionIcon,
+	TickIcon,
+} from './icons';
+
+afterEach(cleanup);
+
+function AppShellTest() {
+	const navItems = [
+		{ label: 'Dashboard', active: true, icon: HomeIcon, href: '#home' },
+		{
+			label: 'Establishments',
+			active: false,
+			icon: FactoryIcon,
+			href: '#establishments',
+		},
+		{
+			label: 'Data and Insights',
+			active: false,
+			icon: ChartIcon,
+			href: '#data',
+		},
+		{
+			label: 'Compliance',
+			active: false,
+			icon: TickIcon,
+			href: '#compliance',
+		},
+		{ isSeperator: true },
```

Comment:
This is a bit of a code smell IMO. Maybe instead of a flat array and using `isSeperator`, the nav items could be an array of arrays? With a horizontal rule being placed between each list.

Reply:
Good thought. The HTML validation test picked up that a HR shouldn't be in a UL, so I'll do that anyway 👍